daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -226,14 +226,30 @@ class CLIConf(arguments: Array[String], stdout: PrintStream, stderr: PrintStream
   implicit def rootNSConverter = org.rogach.scallop.singleArgConverter[RefQName](qnameConvert _)
 
   implicit def fileResourceURIConverter = singleArgConverter[URI]((s: String) => {
-    val file = new File(s)
-    val uri =
-      if (file.isFile()) {
-        Some(file.toURI)
-      } else {
-        Misc.getResourceRelativeOption(s, None)
+    val optURI = None
+      .orElse {
+        val file = new File(s)
+        if (file.isFile) Some(file.toURI)
+        else None
+      }
+      .orElse {
+        if (s.startsWith("/")) {
+          val resource = this.getClass.getResource(s)
+          Option(resource.toURI)
+        } else {
+          None
+        }
+      }
+      .orElse {
+        val resource = this.getClass.getResource("/" + s)
+        if (resource != null) {
+          Logger.log.warn(s"Found relative path on classpath absolutely, did you mean /$s")
```

Comment:
This is not a schema SDW but a logged warning? why isn't this an SDW. I think being able to suppress this may be important for a while.